spec/models/pacbio/well_spec.rb

```diff
@@ -563,4 +563,177 @@
       expect(well.library_type).to eq('Standard')
     end
   end
+
+  context 'sample sheet behaviour' do
```

Comment:
Missing tests for `sequencing_kit_box_barcode_plate_1`, `sequencing_kit_box_barcode_plate_2`

Reply:
The tests are in ther in another section. https://github.com/sanger/traction-service/blob/develop/spec/models/pacbio/well_spec.rb#L544.

I am fine leaving it there.